Sources/BraintreePayPal/BTPayPalVaultBaseRequest.swift

```diff
@@ -19,18 +19,24 @@ import BraintreeCore
     ///   - offerCredit: Optional: Offers PayPal Credit if the customer qualifies. Defaults to `false`.
     ///   - userAuthenticationEmail: Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
     ///   - enablePayPalAppSwitch: Optional: Used to determine if the customer will use the PayPal app switch flow. Defaults to `false`.
+    ///   - recurringBillingPlanType: Optional: Recurring billing plan type, or charge pattern.
+    ///   - userAuthenticationEmail: Optional: User email to initiate a quicker authentication flow in cases where the user has a PayPal Account with the same email.
```

Comment:
Lets make the docstrings match the init. These should be `recurringBillingDetails ` and `recurringBillingPlanType`